server/handlers.go

```diff
@@ -224,8 +224,8 @@ func (s *Server) handleAuthorization(w http.ResponseWriter, r *http.Request) {
 	}
 
 	connectors, e := s.storage.ListConnectors()
-	if e != nil {
-		s.logger.Errorf("Failed to get list of connectors: %v", err)
+	if err != nil {
```

Comment:
Should this not stay `if e != nil {`? Looks like you've fixed the bug twice 😉

Reply:
Well that's embarrassing. I have fixed it in the latest push